docs/book/guides/functional-api/README.md

```diff
@@ -4,7 +4,7 @@ description: Build production ML pipelines from the simple step interface.
 
 # Functional API
 
-The Functional ZenML API is defined by the primitive `@step` and `@pipeline` decorators. These make it easy to quickly take your functional code into a zenml pipeline. If you want more control over the different steps the [Class based API](../high-level-api/)  is the way to go.&#x20;
+The Functional ZenML API is defined by the primitive `@step` and `@pipeline` decorators. These make it easy to quickly take your functional code into a zenml pipeline. If you want more control over the different steps the [Class based API](../class-based-api/)  is the way to go.&#x20;
```

Comment:
```suggestion
The Functional ZenML API is defined by the primitive `@step` and `@pipeline` decorators. These make it easy to quickly take your functional code into a ZenML pipeline. If you want more control over the different steps the [Class based API](../class-based-api/)  is the way to go.&#x20;
```

Reply:
Thanks, I have pushed the fixed version.